### PR TITLE
ci: github action for pushing the docker image

### PR DESCRIPTION
**Pull request by zarazan**
- Add github action for pushing new e2e test images after PRs are merged to master

closes the ojo side of: https://github.com/ojo-network/price-feeder/issues/50

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -45,6 +45,18 @@ install: go.sum
 
 .PHONY: build install
 
+###############################################################################
+##                                  Docker                                   ##
+###############################################################################
+
+docker-build:
+	@docker build -t ghcr.io/ojo-network/price-feeder-ojo .
```
Comment (robert-zaremba) [inline Makefile:53]:
```suggestion
	@DOCKER_BUILDKIT=1 docker build -t ghcr.io/ojo-network/price-feeder-ojo .
```

File: Makefile [hunk 2]
```diff
@@ -45,6 +45,18 @@ install: go.sum
...
+
+docker-push:
+	@docker push ghcr.io/ojo-network/price-feeder-ojo
```
Comment (robert-zaremba) [inline Makefile:56]:
```suggestion
	@DOCKER_BUILDKIT=1 docker push ghcr.io/ojo-network/price-feeder-ojo
```

**Review by robert-zaremba — COMMENTED** [review]
adding suggestion to use `DOCKER_BUILDKIT=1` when available. It has performance improvements and produces better output

**Review by adamewozniak — APPROVED** [review]
nice & simple 👍

File: Makefile (hunk 1 above)
Comment (zarazan) [reply Makefile]: https://github.com/ojo-network/price-feeder/pull/62/commits/dfd10021c388523f55cc310477f9b5eb1e34cb41

Comment (robert-zaremba) [reply Makefile]: you could just accept the suggestion ;)

**Comment by adamewozniak** [comment]
@mergifyio backport umee

**Comment by adamewozniak** [comment]
@mergifyio backport umee

**Pull request merged**